src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -481,7 +481,79 @@ let ``Supports default custom simulator`` () =
     given ["--simulator"; typeof<QuantumSimulator>.FullName; "--use-h"; "false"] |> fails
 
 
-// TODO: Add tests for the "submit" command.
+// Azure Quantum Submission
```

Comment:
do you have tests for when missing required arguments? like missing target/workspace?

Reply:
No, those would be good to add - I'll include them in a separate PR (see comment a the bottom).